pkg/operator/ceph/cluster/cluster.go

```diff
@@ -108,6 +108,7 @@ func (c *cluster) detectCephVersion(image string, timeout time.Duration) (*cephv
 			},
 		},
 	}
+	k8sutil.AddRookVersionLabelToJob(job)
```

Comment:
Before this change are we only setting labels on pod specs? Just wanted to check if we need to worry about existing code overwriting the labels you set here.

Reply:
We have code adding labels to deployments, but the labels are all added before this is called as far as I can tell.